### PR TITLE
Merge develop into main for v0.4.0 release

### PR DESCRIPTION
**Pull request by rtimms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,25 +39,35 @@ my_params_dict = my_params.dict(by_alias=True)
...
 If you want to output the complete JSON schema in order to build a custom tool yourself, you can do so:
-
 ```python
 print(bpx.BPX.schema_json(indent=2))
 ```
 
 According to the `pydantic` docs, the generated schemas are compliant with the specifications: JSON Schema Core, JSON Schema Validation and OpenAPI.
+
+## 📖 Documentation
+API documentation for the `bpx` package can be built locally using [Sphinx](https://www.sphinx-doc.org/en/master/). To build the documentation first [clone the repository](https://github.com/git-guides/git-clone), then run the following command:
```
Comment (ikorotkin) [inline README.md:63]: This didn't work straight away for me ('No module named bpx' error). I just git-cloned the repo (dev branch) and tried the command below.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,26 @@
-# [v0.3.1](https://github.com/pybamm-team/BPX/releases/tag/v0.3.1)
-- Temporarily pin Pydantic version ([#35](https://github.com/pybamm-team/BPX/pull/35))
+# [v0.4.0](https://github.com/FaradayInstitution/BPX/releases/tag/v0.4.0)
+
+- Added five parametrisation examples (two DFN parametrisation examples from About:Energy open-source release, blended electrode definition, user-defined 0th-order hysteresis, and SPM parametrisation). ([#45](https://github.com/FaradayInstitution/BPX/pull/45))
+- Allow user-defined parameters to be added using the field ["Parameterisation"]["User-defined"] ([#44](https://github.com/pybamm-team/BPX/pull/44))
```
Comment (ikorotkin) [inline CHANGELOG.md:4]: There are still a few links to the old (pybamm-team) repo here and below.

**Review by ikorotkin — COMMENTED** [review]
That's a lot of changes to review! :) See a couple of minor comments I left... Otherwise looks good.

File: CHANGELOG.md (hunk 2 above)
Comment (rtimms) [reply CHANGELOG.md:4]: thanks, will update

**Comment by rtimms** [comment]
@ikorotkin thanks, all these things should have already been reviewed once when they were merged into develop, so thanks for the careful re-review!

File: README.md (hunk 1 above)
Comment (rtimms) [reply README.md:63]: you have to install bpx first, will update the instructions

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ This repository features a Pydantic-based parser for JSON files in the BPX forma
 
 To support the new open standard, [About:Energy](https://www.aboutenergy.io/) have supplied two parameters sets for an NMC and LFP cell. The BPX files and associated examples and information can be found on the [A:E BPX Parameterisation repository](https://github.com/About-Energy-OpenSource/About-Energy-BPX-Parameterisation/).
 
-To see how to use BPX with [PyBaMM](https://www.pybamm.org/), check out the [BPX example repository](https://github.com/pybamm-team/bpx-example).
+To see how to use BPX with [PyBaMM](https://www.pybamm.org/), check out the [BPX example repository](https://github.com/FaradayInstitution/bpx-example).
```
Comment (ikorotkin) [inline README.md:11]: Oops, now this link is broken (and a few other links to `bpx-example` below), it should be in `pybamm-team` repo.

File: docs/source/user_guide/getting_started.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ import bpx
 filename = 'path/to/my/file.json'
 my_params = bpx.parse_bpx_file(filename)
 ```
-`my_params` will now be of type `BPX`, which acts like a python dataclass with the same attributes as the BPX format. To obatin example files, see the [A:E BPX Parameterisation repository](https://github.com/About-Energy-OpenSource/About-Energy-BPX-Parameterisation/) or [BPX example repository](https://github.com/pybamm-team/bpx-example).
+`my_params` will now be of type `BPX`, which acts like a python dataclass with the same attributes as the BPX format. To obatin example files, see the [A:E BPX Parameterisation repository](https://github.com/About-Energy-OpenSource/About-Energy-BPX-Parameterisation/) or [BPX example repository](https://github.com/FaradayInstitution/bpx-example).
```
Comment (ikorotkin) [inline docs/source/user_guide/getting_started.md:12]: `bpx-example` repo is in `pybamm-team`

File: README.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ import bpx
 filename = 'path/to/my/file.json'
 my_params = bpx.parse_bpx_file(filename)
 ```
-`my_params` will now be of type `BPX`, which acts like a python dataclass with the same attributes as the BPX format. To obatin example files, see the `examples` folder, the [A:E BPX Parameterisation repository](https://github.com/About-Energy-OpenSource/About-Energy-BPX-Parameterisation/), or the [BPX example repository](https://github.com/pybamm-team/bpx-example).
+`my_params` will now be of type `BPX`, which acts like a python dataclass with the same attributes as the BPX format. To obatin example files, see the `examples` folder, the [A:E BPX Parameterisation repository](https://github.com/About-Energy-OpenSource/About-Energy-BPX-Parameterisation/), or the [BPX example repository](https://github.com/FaradayInstitution/bpx-example).
```
Comment (ikorotkin) [inline README.md:29]: And here as well, should be `pybamm-team`.

**Review by ikorotkin — APPROVED** [review]
It's only the link to `pybamm-team/bpx-example` should be fixed, and we can merge it.

File: README.md (hunk 1 above)
Comment (ikorotkin) [reply README.md]: yeah, I realised that I had installed the previous version of BPX, that's why it probably complained.

**Pull request merged**